pkg/crd/kubermatic/v1/settings.go

```diff
@@ -52,6 +52,7 @@ type SettingSpec struct {
 	DisplayTermsOfService bool           `json:"displayTermsOfService"`
 	EnableDashboard       bool           `json:"enableDashboard"`
 	EnableOIDCKubeconfig  bool           `json:"enableOIDCKubeconfig"`
+	LimitUserProjects     int64          `json:"limitUserProjects"`
```

Comment:
LimitUserProject looks like boolean switch. Can you rename to UserProjectsLimit for example?